metacatalog/test/test_standards_export.py

```diff
@@ -4,25 +4,56 @@
 
 
 from metacatalog import api, ext
+from metacatalog.util.results import ImmutableResultSet
 from ._util import connect
 
 
 def check_entry_iso19115_export(session, config_dict, template_path):
     """
-    Check if ``Entry.standards_export`` produces a ISO 19915 
+    Check if ``Entry.standards_export`` produces an ISO 19915 
     XML ElementTree object for all Entries in the test database.
 
     """
     for entry in api.find_entry(session):
-        xml = entry.standards_export(config_dict, template_path)
+        xml_etree = entry.standards_export(config_dict, template_path)
 
-        assert isinstance(xml, ET.ElementTree)
+        assert isinstance(xml_etree, ET.ElementTree)
 
     return True
 
 
+def check_api_iso19115_export(session, config_dict, path):
+    """
+    Check if ``api.catalog.create_iso19115_xml`` creates an ISO
+    19115 .xml file for all Entries in the test database.
+    Also checks that the XML ElementTree object is returned if
+    no path is given to ``create_iso19115_xml``.
+    
+    """    
+    for entry in api.find_entry(session):
+        # create xml
+        api.catalog.create_iso19115_xml(session, id_or_uuid=entry.id, config_dict=config_dict, path=path)
+
+        # get ImmutableResultSet uuid to open file
+        irs_uuid = ImmutableResultSet(entry).uuid
+
+        # .xml filenames are generated from ImmutableResultSet.uuid
+        with open(f"{path}/iso19115_{irs_uuid}.xml") as f:
+            xml_str = f.read()
+            
+            # check expected start of the xml string
+            assert xml_str.startswith("<gmi:MI_Metadata")
+
```

Comment:
extra assert to test if title is in xml_str

```python
assert entry.title in xml_str, f'[{i + 1}] entry_id = {entry.id}: Title is not contained'
```

Reply:
If a base group exists in the ImmutableResultSet of an Entry, the group title is included and NOT the title of the individual Entry.  
I will check for `ImmutableResultSet.uuid` instead.

Reply:
makes sense